src/Sylius/Bundle/CartBundle/Provider/CartProvider.php

```diff
@@ -62,27 +62,26 @@ class CartProvider implements CartProviderInterface
     protected $eventDispatcher;
 
     /**
-     * Cart.
-     *
-     * @var CartInterface
-     */
-    protected $cart;
-
-    /**
      * Constructor.
      *
      * @param CartContextInterface     $context
      * @param ObjectManager            $manager
-     * @param FactoryInterface $cartFactory
-     * @param RepositoryInterface      $repository
+     * @param FactoryInterface         $cartFactory
+     * @param RepositoryInterface      $cartRepository
      * @param EventDispatcherInterface $eventDispatcher
      */
-    public function __construct(CartContextInterface $context, ObjectManager $manager, FactoryInterface $cartFactory, RepositoryInterface $repository, EventDispatcherInterface $eventDispatcher)
+    public function __construct(
+        CartContextInterface $context,
+        ObjectManager $manager,
+        FactoryInterface $cartFactory,
+        RepositoryInterface $cartRepository,
+        EventDispatcherInterface $eventDispatcher
+    )
     {
         $this->context = $context;
         $this->manager = $manager;
```

Comment:
`$this->cartManager = $cartManager` - to be consistent with other cart* variables?